components/theme.ts

```diff
@@ -45,14 +42,6 @@ export interface ThemedProps {
 
 const ThemeContext = EmotionThemeContext as React.Context<Theme>;
 
-// Hack : override the default value of the context used for theme by emotion
-// As we can't pass a default value when it is constructed
-// This allows outside tests to get the mocked theme value without specifiying a theme provider
-if (process.env.NODE_ENV === 'test') {
-  (ThemeContext as any)['_currentValue'] = mockedTheme;
-  (ThemeContext as any)['_currentValue2'] = mockedTheme;
-}
```

Comment:
I guess this has been introduced to avoid passing the context in each tests located in SC or SQ ("outside tests"), not only for SUC tests.

Are your SC tests green when linked to this version of SUC ? Sadly I've trouble to test it on the SQ side cause we've not implemented yet the new init process you introduced lately, and it is polluting my tests.

Reply:
True! Good catch!
We need to update our "SetupTestEnvironment.js" file with something like this:
```js
const theme = require('../../src/app/theme');
const { default: ThemeContext } = require('sonar-ui-common/components/theme');

ThemeContext['_currentValue'] = theme;
ThemeContext['_currentValue2'] = theme;
```

I'll update the changelog and add this to the readme.

Reply:
Please link the SC PR in this one. We'll try to follow you folks asap.

Reply:
I didn't do a PR on SC yet, since this is not released, but I'll ping you on it once done